contracts/eos/BancorConverter/BancorConverter.hpp

```diff
@@ -302,28 +369,34 @@ CONTRACT BancorConverter : public eosio::contract { /*! \endcond */
          * @param quantity - the quantity for the transfer
          * @param memo - the memo for the transfer
          */
-        //[[eosio::on_notify("*::transfer")]]
+        [[eosio::on_notify("*::transfer")]]
         void on_transfer(name from, name to, asset quantity, string memo);
 
+        [[eosio::action]]
+        void migrate( const set<symbol_code> converters );
+
+        [[eosio::action]]
+        void delmigrate( const set<symbol_code> converters );
+
         /*! \cond DOCS_EXCLUDE */
-        typedef eosio::multi_index<"settings"_n, settings_t> settings;
+        typedef eosio::singleton<"settings"_n, settings_params> settings;
```

Comment:
settings_t?

Reply:
I had changed the `struct` to `settings_params`, but we can change it back to `settings_t`

Reply:
changed it back to `BancorConverter::settings_t`